src/resolution_functions/instrument_data/arcs.yaml

```diff
@@ -34,8 +33,8 @@ version:
                 Fermi:
                     distance: 11.61  # m
                     aperture_distance: 9.342  # m
-        configurations: &configurations
-            chopper_package:
+        configurations:
+            chopper_package: &configurations
```

Comment:
What's the reasoning to move this rather than rename it as well?

Reply:
laziness, lack of attention to detail ...

Reply:
"the human condition"